x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/transform_function_selector.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiCard, EuiFlexGroup, EuiFlexItem, EuiIcon, EuiSpacer } from '@elastic/eui';
+import { TRANSFORM_FUNCTION, TransformFunction } from '../../../../../../common/constants';
+
+interface TransformFunctionSelectorProps {
+  selectedFunction: TransformFunction;
+  onChange: (update: TransformFunction) => void;
+}
+
+export const TransformFunctionSelector: FC<TransformFunctionSelectorProps> = ({
+  selectedFunction,
+  onChange,
+}) => {
+  const transformFunctions = [
+    {
+      name: TRANSFORM_FUNCTION.PIVOT,
+      helpText: i18n.translate('xpack.transform.stepDefineForm.pivotHelperText', {
+        defaultMessage: 'Pivot function',
+      }),
+      icon: 'aggregate',
+      title: i18n.translate('xpack.transform.stepDefineForm.pivotLabel', {
+        defaultMessage: 'Pivot',
+      }),
+    },
+    {
+      name: TRANSFORM_FUNCTION.LATEST,
+      helpText: i18n.translate('xpack.transform.stepDefineForm.latestHelperText', {
+        defaultMessage: 'Latest function',
+      }),
+      icon: 'clock',
+      title: i18n.translate('xpack.transform.stepDefineForm.latestLabel', {
+        defaultMessage: 'Latest',
+      }),
+    },
+  ];
+
+  return (
+    <>
+      <EuiFlexGroup gutterSize="m" data-test-subj="transformFunctionSelection">
+        {transformFunctions.map(({ helpText, icon, name, title }) => (
+          <EuiFlexItem key={name} grow={1}>
+            <EuiCard
```

Comment:
Is it possible to give the cards a fixed width? With only two cards, they look a bit odd to me taking up the full width of the panel.

Reply:
Changed in https://github.com/elastic/kibana/pull/85784/commits/3c15fe32e1765bba64ef3315464c72a6489e11cb